### PR TITLE
starting the loop for energy flux

### PR DESCRIPTION
**Pull request by skadam02**
Pull request for comments

### CODE REVIEW

File: mixed_pond_experimental/energy_flux.py [hunk 1]
```diff
@@ -68,20 +70,25 @@ def read_dataline(day_argue):
...
-    R_s = 2.2*((180*lambda_value)/pi)**(-0.97)
     
+
+    if lambda_value == 0:
+            R_s = 0
+    else:
+            R_s = 2.2*((180*lambda_value)/pi)**(-0.97)
+        
+
     W_z = wind_speed * 3.6
```
Comment (drewmresnick) [inline mixed_pond_experimental/energy_flux.py:83]: Where does th 3.6 value come from? The equation in Culberson & Piedrahita it does not have this so I may be missing something?

Comment (skadam02) [reply mixed_pond_experimental/energy_flux.py:83]: Oh it's to convert the wind speed from m/s to km/hr i think (per previous code) But there is another place where we need the units to be m/hr and another where it's km/hr..could you please check and let me know i may have messed up that conversion

File: mixed_pond_experimental/energy_flux.py [hunk 2]
```diff
@@ -68,20 +70,25 @@ def read_dataline(day_argue):
...
     
     R= R_s *(1-0.08 * W_z)
     
-    phi_s = int(float(daily_data[8+hour_period]))   #8th column plus number of hour gets us sol rad for that hour units watts/m square
-    #have to convert string to integer and then to float everytime is there a better way to handle this? (question for drew)
+    index = srad_names[hour_period-1]  #picking the hour index for 3 hour window units watts/m square
+    
+    phi_s = daily_data[index]
```
Comment (drewmresnick) [inline mixed_pond_experimental/energy_flux.py:89]: Should be in kJ/m2/hr. Add this in a comment for us to keep track?

File: mixed_pond_experimental/energy_flux.py [hunk 3]
```diff
@@ -133,21 +139,20 @@ def calculate_phi_e(T_wk,day_argue, hour_period):
...
+    e_s = 25.374 * math.exp(17.62 - 5271/T_wc)
 
 # e_a, water vapor pressure above the pond surface; unit mmHg
-    e_a= 6.1078 math.exp( (17.269 * T_d) / (237.3 + T_d) )
-          #where,
-#e is saturated vapor pressure in millibars
-#T is temperature in degrees C
-    
-    phi_e = N* W_2 * (e_s- e_a)
+    e_a= 610.78 * math.exp(17.269 * ((T_d -273.16) / (T_d - 35.86)))
```
Comment (drewmresnick) [inline mixed_pond_experimental/energy_flux.py:152]: Remind me where did we get this constant for Rh?

File: mixed_pond_experimental/energy_flux.py [hunk 4]
```diff
@@ -156,12 +161,47 @@ def calculate_phi_c(T_wk, day_argue, hour_period):
     daily_data = read_dataline(day_argue)
     wind_speed = daily_data['wind_speed_2m']
 
-    W = wind_speed * 3.6 # Convert ms-1 to Kmhr-1 #check units for all variables ask drew
+    W = float(wind_speed * 3.6) # Convert ms-1 to Kmhr-1 #check units for all variables ask drew
```
Comment (drewmresnick) [inline mixed_pond_experimental/energy_flux.py:164]: In culberson and Piedrahita it says the wind speed should be in m/s here

File: mixed_pond_experimental/energy_flux.py (hunk 1 above)
Comment (drewmresnick) [reply mixed_pond_experimental/energy_flux.py:83]: in C&P it shows the equation the have wind speed = wind velocity at height z meters above pond surface (m/s). (on page 254 I see this). Where did we find that it should be in km/hr?

I realize I did not look into this carefully enough the first go around sorry about that.

**Pull request merged**